packages/common/src/services/auth/hedgehog.ts

```diff
@@ -89,5 +89,16 @@ export const createHedgehog = ({
     )
   }
 
+  // TODO (PAY-3479): This is temporary until hedgehog is fully moved out of libs
+  // Listen for changes to libs instance and update our local wallet
+  window.addEventListener('hedgehogWalletUpdate', () => {
```

Comment:
I decided to remove the manual updates pushed from sagas in favor of adding an event mechanism to the libs instance of Hedgehog. 
There are some deeply-nested places where entropy gets generated or updated that are very hard to wire up. But letting the second instance of Hedgehog just watch for an event from the libs instance and refreshing its entropy in response turns out to be a perfectly functional and safe pattern.